panoptes_client/tests/test_subject_set.py

```diff
@@ -33,6 +33,7 @@ def test_create(self):
                 json={
                     'subject_sets': {
                         'display_name': 'Name',
+                        'metadata': {},
```

Comment:
nitpicky, but we can probably test the case where we want metadata to be updated. 

something similar to this but with 
`subject_set.metadata['indexField'] = some index field` 

to test the case that is mentioned in issue https://github.com/zooniverse/panoptes-cli/issues/256 broken in cli. 

Given the state of python client (and the fact that you and Toyosi have mentioned that you all have tested extensively) , this test I guess is not necessary but would be a nice to have.

Reply:
I'm crap at writing tests.  If you and Toyosi can figure out how to write an appropriate test for the with and without metadata cases, please do go ahead and add.  However, I don't consider that a requirement for merging this PR -- the current test is basically a unit test, and the adjustment I made was to reflect how existing resources are loaded.